[2mdn.net] HTTPSConnectionPool(host='2mdn.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7510e360>: Failed to resolve '2mdn.net' ([Errno -5] No address associated with hostname)"))
[a-msedge.net] HTTPSConnectionPool(host='a-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7510dc10>: Failed to resolve 'a-msedge.net' ([Errno -5] No address associated with hostname)"))
[a2z.com] HTTPSConnectionPool(host='a2z.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7510f980>: Failed to resolve 'a2z.com' ([Errno -5] No address associated with hostname)"))
[aaplimg.com] HTTPSConnectionPool(host='aaplimg.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa6f660cb0>: Failed to resolve 'aaplimg.com' ([Errno -5] No address associated with hostname)"))
[adnxs.com] HTTPSConnectionPool(host='adnxs.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7510f290>: Failed to resolve 'adnxs.com' ([Errno -5] No address associated with hostname)"))
[aiv-cdn.net] HTTPSConnectionPool(host='aiv-cdn.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f09a90>: Failed to resolve 'aiv-cdn.net' ([Errno -5] No address associated with hostname)"))
[akadns.net] HTTPSConnectionPool(host='akadns.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f2c4d0>: Failed to resolve 'akadns.net' ([Errno -5] No address associated with hostname)"))
[akamai.net] HTTPSConnectionPool(host='akamai.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f09af0>: Failed to resolve 'akamai.net' ([Errno -5] No address associated with hostname)"))
[akamaiedge.net] HTTPSConnectionPool(host='akamaiedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f0bc50>: Failed to resolve 'akamaiedge.net' ([Errno -5] No address associated with hostname)"))
[akamaihd.net] HTTPSConnectionPool(host='akamaihd.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f2d220>: Failed to resolve 'akamaihd.net' ([Errno -5] No address associated with hostname)"))
[akamaized.net] HTTPSConnectionPool(host='akamaized.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f0ab70>: Failed to resolve 'akamaized.net' ([Errno -5] No address associated with hostname)"))
[amazon-adsystem.com] HTTPSConnectionPool(host='amazon-adsystem.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f2c380>: Failed to resolve 'amazon-adsystem.com' ([Errno -5] No address associated with hostname)"))
[apple-dns.net] HTTPSConnectionPool(host='apple-dns.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74bb4ef0>: Failed to resolve 'apple-dns.net' ([Errno -5] No address associated with hostname)"))
[appsflyersdk.com] HTTPSConnectionPool(host='appsflyersdk.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74ad8e60>: Failed to resolve 'appsflyersdk.com' ([Errno -5] No address associated with hostname)"))
[azureedge.net] HTTPSConnectionPool(host='azureedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa744ee300>: Failed to resolve 'azureedge.net' ([Errno -5] No address associated with hostname)"))
[azurewebsites.net] HTTPSConnectionPool(host='azurewebsites.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74ad34a0>: Failed to resolve 'azurewebsites.net' ([Errno -5] No address associated with hostname)"))
[b-msedge.net] HTTPSConnectionPool(host='b-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa744c3ec0>: Failed to resolve 'b-msedge.net' ([Errno -5] No address associated with hostname)"))
[bytefcdn-oversea.com] HTTPSConnectionPool(host='bytefcdn-oversea.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f11220>: Failed to resolve 'bytefcdn-oversea.com' ([Errno -5] No address associated with hostname)"))
[bytefcdn-ttpeu.com] HTTPSConnectionPool(host='bytefcdn-ttpeu.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f12450>: Failed to resolve 'bytefcdn-ttpeu.com' ([Errno -5] No address associated with hostname)"))
[cdn-apple.com] HTTPSConnectionPool(host='cdn-apple.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73c5ee70>: Failed to resolve 'cdn-apple.com' ([Errno -5] No address associated with hostname)"))
[cdn77.org] HTTPSConnectionPool(host='cdn77.org', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f10fe0>: Failed to resolve 'cdn77.org' ([Errno -5] No address associated with hostname)"))
[cdninstagram.com] HTTPSConnectionPool(host='cdninstagram.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f13530>: Failed to resolve 'cdninstagram.com' ([Errno -5] No address associated with hostname)"))
[cloudapp.net] HTTPSConnectionPool(host='cloudapp.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f13200>: Failed to resolve 'cloudapp.net' ([Errno -5] No address associated with hostname)"))
[cloudfront.net] HTTPSConnectionPool(host='cloudfront.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f2bb30>: Failed to resolve 'cloudfront.net' ([Errno -5] No address associated with hostname)"))
[demdex.net] HTTPSConnectionPool(host='demdex.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa742af020>: Failed to resolve 'demdex.net' ([Errno -5] No address associated with hostname)"))
[dnsowl.com] HTTPSConnectionPool(host='dnsowl.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73c5f530>: Failed to resolve 'dnsowl.com' ([Errno -5] No address associated with hostname)"))
[domaincontrol.com] HTTPSConnectionPool(host='domaincontrol.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7faa7462da30>: Failed to establish a new connection: [Errno 111] Connection refused'))
[dual-s-msedge.net] HTTPSConnectionPool(host='dual-s-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74911df0>: Failed to resolve 'dual-s-msedge.net' ([Errno -5] No address associated with hostname)"))
[edgekey.net] HTTPSConnectionPool(host='edgekey.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa740d78c0>: Failed to resolve 'edgekey.net' ([Errno -5] No address associated with hostname)"))
[edgesuite.net] HTTPSConnectionPool(host='edgesuite.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa740d61e0>: Failed to resolve 'edgesuite.net' ([Errno -5] No address associated with hostname)"))
[ggpht.com] HTTPSConnectionPool(host='ggpht.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74355d00>: Failed to resolve 'ggpht.com' ([Errno -5] No address associated with hostname)"))
[gtld-servers.net] HTTPSConnectionPool(host='gtld-servers.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7495c8c0>: Failed to resolve 'gtld-servers.net' ([Errno -5] No address associated with hostname)"))
[gvt1.com] HTTPSConnectionPool(host='gvt1.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7448b830>: Failed to resolve 'gvt1.com' ([Errno -5] No address associated with hostname)"))
[gvt2.com] HTTPSConnectionPool(host='gvt2.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74489df0>: Failed to resolve 'gvt2.com' ([Errno -5] No address associated with hostname)"))
[l-msedge.net] HTTPSConnectionPool(host='l-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73e411c0>: Failed to resolve 'l-msedge.net' ([Errno -5] No address associated with hostname)"))
[macromedia.com] HTTPSConnectionPool(host='macromedia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7faa741726c0>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[mangosip.ru] HTTPSConnectionPool(host='mangosip.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7faa74b3d160>, 'Connection to mangosip.ru timed out. (connect timeout=None)'))
[microsoftonline.com] HTTPSConnectionPool(host='microsoftonline.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa744ef530>: Failed to resolve 'microsoftonline.com' ([Errno -5] No address associated with hostname)"))
[miit.gov.cn] HTTPSConnectionPool(host='miit.gov.cn', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa744efbf0>: Failed to resolve 'miit.gov.cn' ([Errno -5] No address associated with hostname)"))
[msedge.net] HTTPSConnectionPool(host='msedge.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7faa746857c0>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[msftncsi.com] HTTPSConnectionPool(host='msftncsi.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa746843e0>: Failed to resolve 'msftncsi.com' ([Errno -5] No address associated with hostname)"))
[netflix.net] HTTPSConnectionPool(host='netflix.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7faa74906780>, 'Connection to netflix.net timed out. (connect timeout=None)'))
[nflxso.net] HTTPSConnectionPool(host='nflxso.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74905e80>: Failed to resolve 'nflxso.net' ([Errno -5] No address associated with hostname)"))
[nr-data.net] HTTPSConnectionPool(host='nr-data.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74329c40>: Failed to resolve 'nr-data.net' ([Errno -5] No address associated with hostname)"))
[o365filtering.com] HTTPSConnectionPool(host='o365filtering.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74657740>: Failed to resolve 'o365filtering.com' ([Errno -5] No address associated with hostname)"))
[office.net] HTTPSConnectionPool(host='office.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa7495e7b0>: Failed to resolve 'office.net' ([Errno -5] No address associated with hostname)"))
[omtrdc.net] HTTPSConnectionPool(host='omtrdc.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74955dc0>: Failed to resolve 'omtrdc.net' ([Errno -5] No address associated with hostname)"))
[rbxcdn.com] HTTPSConnectionPool(host='rbxcdn.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74f45af0>: Failed to resolve 'rbxcdn.com' ([Errno -5] No address associated with hostname)"))
[rocket-cdn.com] HTTPSConnectionPool(host='rocket-cdn.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa743544d0>: Failed to resolve 'rocket-cdn.com' ([Errno -5] No address associated with hostname)"))
[root-servers.net] HTTPSConnectionPool(host='root-servers.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa743550d0>: Failed to resolve 'root-servers.net' ([Errno -5] No address associated with hostname)"))
[rubiconproject.com] ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
[s-msedge.net] HTTPSConnectionPool(host='s-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74357b30>: Failed to resolve 's-msedge.net' ([Errno -5] No address associated with hostname)"))
[spo-msedge.net] HTTPSConnectionPool(host='spo-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73d70e60>: Failed to resolve 'spo-msedge.net' ([Errno -5] No address associated with hostname)"))
[t-msedge.net] HTTPSConnectionPool(host='t-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73d713a0>: Failed to resolve 't-msedge.net' ([Errno -5] No address associated with hostname)"))
[tfbnw.net] HTTPSConnectionPool(host='tfbnw.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa73f13050>: Failed to resolve 'tfbnw.net' ([Errno -5] No address associated with hostname)"))
[tiktokcdn-us.com] HTTPSConnectionPool(host='tiktokcdn-us.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74296840>: Failed to resolve 'tiktokcdn-us.com' ([Errno -5] No address associated with hostname)"))
[tiktokcdn.com] HTTPSConnectionPool(host='tiktokcdn.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74bb57c0>: Failed to resolve 'tiktokcdn.com' ([Errno -5] No address associated with hostname)"))
[trafficmanager.net] HTTPSConnectionPool(host='trafficmanager.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74296570>: Failed to resolve 'trafficmanager.net' ([Errno -5] No address associated with hostname)"))
[trbcdn.net] HTTPSConnectionPool(host='trbcdn.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'trbcdn.net'. (_ssl.c:1000)")))
[ttlivecdn.com] HTTPSConnectionPool(host='ttlivecdn.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74bb7aa0>: Failed to resolve 'ttlivecdn.com' ([Errno -5] No address associated with hostname)"))
[wac-msedge.net] HTTPSConnectionPool(host='wac-msedge.net', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74d5eff0>: Failed to resolve 'wac-msedge.net' ([Errno -5] No address associated with hostname)"))
[windowsupdate.com] HTTPSConnectionPool(host='windowsupdate.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa74df3e90>: Failed to resolve 'windowsupdate.com' ([Errno -5] No address associated with hostname)"))
[worldfcdn2.com] HTTPSConnectionPool(host='worldfcdn2.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa744c2360>: Failed to resolve 'worldfcdn2.com' ([Errno -5] No address associated with hostname)"))
[wsdvs.com] HTTPSConnectionPool(host='wsdvs.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa746869c0>: Failed to resolve 'wsdvs.com' ([Errno -5] No address associated with hostname)"))
[ytimg.com] HTTPSConnectionPool(host='ytimg.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7faa741e0b60>: Failed to resolve 'ytimg.com' ([Errno -5] No address associated with hostname)"))
